style.css

```diff
@@ -395,112 +395,6 @@ form > button:hover {
   background: #2230d2;
 }
 
-@media screen and (min-width: 728px) {
-  #desk-list {
-    display: flex;
-    min-width: 200px;
-  }
-
-  #Union-menu {
-    display: none;
-  }
-
-  h1 {
-    font-size: 44px;
-    max-width: 400px;
-    font-family: var(--mainF);
-    font-weight: 700px;
-  }
-
-  main {
-    padding: 200px 100px 150px 100px;
-  }
-
-  p {
-    font-size: 20px;
-  }
-
-  #projects {
-    padding: 100px 50px;
-  }
-
-  .card {
-    max-width: calc(100% - 100px);
-    flex-direction: row;
-  }
-
-  .card-img {
-    width: 55%;
-    padding-right: 20px;
-    margin: auto;
-  }
-
-  .details {
-    width: 45%;
-    padding-right: 20px;
-    padding-top: 25px;
-  }
-
-  #projects :nth-child(2n) > .details {
-    order: 1;
-  }
-
-  #projects :nth-child(2n) > .card-img {
-    order: 2;
-  }
-
-  #about-me {
-    flex-direction: row;
-  }
-
-  #about-myself {
-    width: 40%;
-  }
-
-  #lfs-section {
-    margin: 0;
-    padding: 0 30px;
-    max-width: 100%;
-  }
-
-  #lfs-section > li {
-    margin: 0 0 20px 0;
-  }
-
-  #about-myself-2 {
-    width: 60%;
-  }
-
-  #languages > ul {
-    flex-direction: row !important;
-    justify-content: flex-start;
-  }
-
-  #languages > ul > li {
-    flex-direction: column !important;
-    margin: 0 10px 10px 0 !important;
-    width: 120px;
-    height: 120px !important;
-    justify-content: space-evenly;
-  }
-
-  #languages > ul > li > h6 {
-    margin: 0 !important;
-  }
-
-  #contact {
-    align-items: center;
-  }
-
-  #contact > div {
-    max-width: 700px;
-  }
-
-  #contact > form {
-    max-width: 500px;
-  }
-}
-
 @media screen and (min-width: 1200px) {
   nav {
     padding: 15px 150px;
```

Comment:
- [ ] Please make sure your design sticks as much as possible to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Your header is not the same as in Figma design, in the desktop version the hamburger menu is not visible and the menu display, please kindly fix it 🙃 

> |     | Your design                      | Figma design                     |     |
> | --- | ----------------------------------- | ------------------------------------ | --- |
> |       | <img width="954" alt="Screen Shot 2022-09-29 at 12 18 41" src="https://user-images.githubusercontent.com/74501523/193008974-a8564633-0f52-4a46-9f09-c9fa423c733d.png"> | <img width="929" alt="Screen Shot 2022-09-29 at 12 19 59" src="https://user-images.githubusercontent.com/74501523/193009132-2f9a0a24-7628-462a-8131-f1704e630fab.png"> |

- [ ] Your works section should stick as much as possible to the Figma design, you should use both Flexbox and Grid in order to layout the elements on the page, please kindly fix it 😉 

> |     | Your design                      | Figma design                     |     |
> | --- | ----------------------------------- | ------------------------------------ | --- |
> |       | <img width="637" alt="Screen Shot 2022-09-29 at 12 20 55" src="https://user-images.githubusercontent.com/74501523/193010414-0de8f35d-0017-447f-9b69-d490b59cf4be.png"> | <img width="407" alt="Screen Shot 2022-09-29 at 12 21 37" src="https://user-images.githubusercontent.com/74501523/193010533-b8ef711c-1190-431f-8847-e2b5a602f5e1.png"> |

- [ ] Your `About` section should also stick as much as possible to the Figma design, please kindly fix it as well 😉 😉 

> |     | Your design                      | Figma design                     |     |
> | --- | ----------------------------------- | ------------------------------------ | --- |
> |       | <img width="638" alt="Screen Shot 2022-09-29 at 12 23 25" src="https://user-images.githubusercontent.com/74501523/193011209-31efcd35-d560-4667-82f2-a8abac151fc4.png"> | <img width="881" alt="Screen Shot 2022-09-29 at 12 24 12" src="https://user-images.githubusercontent.com/74501523/193011285-3f1b2060-3c9b-4362-a5e7-1b4e1cfebb39.png">|

- [ ] I noticed that you don't have the contact form section on your website, for both desktop and mobile versions, please kindly add it 🙃

<img width="881" alt="Screen Shot 2022-09-29 at 12 24 36" src="https://user-images.githubusercontent.com/74501523/193011836-c6211637-9b40-48fd-949c-91c20fced8ba.png">

- [ ] When you finish the desktop version you will ensure that your mobile version design sticks to the Figma design as much as possible. The importance of reproducing the exact design is to ensure that in the future if the customer gives you a design, you will be able to reproduce the exact design and meet all the customer's needs 😁 😁